packages/gatsby-remark-prismjs/src/__tests__/load-prism-language.js

```diff
@@ -41,4 +41,32 @@ describe(`load prism language`, () => {
     expect(Prism.languages).toHaveProperty(requiredLanguage)
     expect(languagesAfterLoaded.length).toBe(languagesBeforeLoaded.length + 2)
   })
+
+  it(`loads languages when referred by their aliases (set as a String in components)`, () => {
```

Comment:
Sorry about these test names 🤦‍♂️ 

If you think of better ones, please recommend!